cluster-state-service/internal/features/e2einstancesteps/get_instance_steps.go

```diff
@@ -66,6 +67,26 @@ func init() {
 		cssContainerInstanceList = append(cssContainerInstanceList, *cssInstance)
 	})
 
+	When(`^I get instance where the task was started$`, func() {
```

Comment:
I know this is a get call, but should this be part of the stream_instances_steps.go file? We don't use it in the get tests right?